src/cli/node/with_node.rs

```diff
@@ -580,6 +586,21 @@ pub enum Command {
         #[structopt(name = "peers")]
         peers: Vec<SocketAddr>,
     },
+    #[structopt(name = "clearPeers", about = "Clear all peers from the buckets. These will leave your node without any connections")]
```

Comment:
The outbounds and inbounds are not cleared, so technically you still have connections. It only clear the buckets